stablehlo/tests/stablehlo_refine_shapes.mlir

```diff
@@ -424,6 +424,54 @@ func.func @eval_slice() -> tensor<2xi64> {
 
 // -----
 
+// CHECK-LABEL: func @eval_slice_unit_prefix
+func.func @eval_slice_unit_prefix() -> (tensor<1x1x1x2xi64>, tensor<1x1x1x2xi64>, tensor<1x1x1x2xi64>) {
```

Comment:
Could you add a PR description with a bit of information on the feature change / use case? Will make it easier to review / for anyone to view PR and get a high level understanding.